bugbug/models/spambug.py

```diff
@@ -87,5 +89,33 @@ def get_labels(self):
 
         return classes, [0, 1]
 
+    def items_gen(self, classes):
```

Comment:
It is overly complex for this case, as we already know commit_data is false here, so we can just get rid of all that code